Update main readme for the upcoming 5.0.0 release (#1580)
jplag/JPlag: README.md

[![Java Version](https://img.shields.io/badge/java-SE%2021-yellowgreen)](#download-and-installation)

> JPlag finds pairwise similarities among a set of multiple programs. It can reliably detect software plagiarism and collusion in software development, even when obfuscated. JPlag currently supports various programming and modeling languages. Moreover, it has rudimentary support for natural language.

Reviewer: ```suggestion
JPlag finds pairwise similarities among a set of multiple programs. It can reliably detect software plagiarism and collusion in software development, even when obfuscated. JPlag supports a large number of programming and modeling languages. Moreover, it has rudimentary support for natural language.
```

> * [JPlag Demo](https://jplag.github.io/Demo/)

Reviewer: ```suggestion
* 📈 [JPlag Demo](https://jplag.github.io/Demo/)
```

> * [JPlag on Helmholtz RSD](https://helmholtz.software/software/jplag)

Reviewer: ```suggestion
* 🏛️ [JPlag on Helmholtz RSD](https://helmholtz.software/software/jplag)
```

> * [Give us Feedback in a **short survey**](https://docs.google.com/forms/d/e/1FAIpQLSckqUlXhIlJ-H2jtu2VmGf_mJt4hcnHXaDlwhpUL3XG1I8UYw/viewform?usp=sf_link)

Reviewer: ```suggestion
* 🤩 [Give us Feedback in a **short survey**](https://docs.google.com/forms/d/e/1FAIpQLSckqUlXhIlJ-H2jtu2VmGf_mJt4hcnHXaDlwhpUL3XG1I8UYw/viewform?usp=sf_link)
```
